docs/tutorials/kafka-migration.rst

```diff
@@ -0,0 +1,368 @@
+* Use :doc:`Aiven command line interface </docs/tools/cli>`.
+
+
+Migrate access control list
+---------------------------
+
+Apache Kafka Access Control Lists (ACLs) define how various users can interact with the topics and schemas. To migrate ACLs we recommend to extract the ACL definition from the source Apache Kafka cluster and recreate them in the target cluster. 
+
+If the target of the migration is Aiven for Apache Kafka you can define the ACLs with:
+
+* The `Aiven Console <https://console.aiven.io/>`__, service page, **Access Control Lists (ACL)** tab.
+* The dedicated `Aiven REST API <https://api.aiven.io/doc/#tag/Service:_Kafka/operation/ServiceKafkaAclAdd>`__.
+* The dedicated :doc:`Aiven CLI command </docs/tools/cli/service/acl>`.
+
+
+Change clients settings
+-----------------------
+
+After the replication flow is running, with the schemas and ACLs are in place, you can then start point producers and consumers to the target Apache Kafka cluster. 
+
+.. Warning::
+
+  To avoid any Apache Kafka messages to be lost during the asynchronous migration performed with MirrorMaker 2, it is suggested to stop the producers, check that both the consumer lag in the source system and the MirrorMaker 2 replication lag is ``0`` and then point producers and consumers to the target Apache Kafka cluster. 
+  
+  The :ref:`migration process <tutorial_kafka_migration_migration_process>` provides a detailed series of steps to follow.
+
+If the target of the migration is Aiven for Apache Kafka, you can find how to download the required certificates and connect as producer or consumer in the :doc:`dedicated documentation </docs/products/kafka/howto/list-code-samples>`.
+
+
+
+.. _tutorial_kafka_migration_migration_process:
+
+Complete migration process
+--------------------------
+
+The following diagram showcases all the steps included in an Apache Kafka migration process.
+
+.. mermaid::
+
+    graph TD;
+
+        id1(Start Migration process)-->id2(Create target Apache Kafka service);
+        id2-->id3(Create Apache Kafka topics in the target cluster);
+        id3-->id4(Create Apache Kafka schemas in the target cluster);
+        id4-->id5(Create Apache Kafka ACLs in the target cluster);
+        id5-->id6(Create source Apache Kafka cluster endpoint);
+        id6-->id7(Create Apache Kafka MirrorMaker 2 service);
+        id7-->id8(Define MirrorMaker 2 aliases for source and target Apache Kafka clusters);
+        id8-->id9(Stop Apache Kafka producers on the original Apache Kafka cluster);
+        id9-->id10(Wait for consumer lag to go to 0 in the original Apache Kafka cluster);
+        id10-->id11(Wait for MirrorMaker 2 replication lag to go to 0);
+        id11-->id12(Stop consumers in the source Apache Kafka cluster);
+        id12-->id13(Point consumers to the target Apache Kafka cluster and start them);
+        id13-->id14(Point producers to the target Apache Kafka cluster and start them);
+        id14-->id15(End Migration process);
```

Comment:
What are the reader's next steps? 

What should they check on after the migration is done, to ensure that things went as planned?

Reply:
Added a final brief section on the four main actions to check (ACLs, schemas, topics&data, producers and consumers)